src/BetweenTheBubbles.tbasic

```diff
@@ -14,6 +14,15 @@ uses "tbgl"
 #include "screen_definitions/title.tbasicu"
 #include "screen_definitions/game.tbasicu"
 #include "screen_definitions/gameover.tbasicu"
+#include ONCE "screen_definitions/language.tbasicu"
+
+#Region I18N_Tweaking
+  Const AllLanguages_filename as String Value = "Translations" ' Translations.AllLanguages.xml
```

Comment:
Dear Sebastian,

thanks a lot for this PR.

I would suggest to USE_UPPER_CASE for constants, also, to start string constants with $.

So, for example, instead of:
`Const AllLanguages_filename as String Value = "Translations"`

You can do:
`$ALL_LANGUAGES_FILENAME = "Translations"`

It is a convention in thinBASIC language.

Reply:
So _Const_ keyword is not required, right  ?
Im' OK with the suggestion. Remark : [https://www.thinbasic.com/public/products/thinBasic/help/html/const.htm](url) do not show this, neither the example provided.

Reply:
Yes - it can be confusing, I understand - `const` is general purpose way to build a constant.

$CONSTANT = "a" is a shortcut for string constants, %CONSTANT is a shortcut for integer constants.

`Const $CONSTANT as string value = ` is perfectly valid, however, quite long way to express.

Reply:
> a shortcut 

ok!